changelogs/fragments/219-droplet-firewall-changed-reporting.yaml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - digital_ocean_droplet - fix reporting of changed state when ``firewall`` argument is present
```

Comment:
This is missing a reference at the end, e.g., "... is present (link to this PR)." -- please look at the [documentation](https://docs.ansible.com/ansible/latest/community/development_process.html#changelog-fragment-entry-format).